src/scss/blocks/_figure.scss

```diff
@@ -1,35 +1,38 @@
 /// COMPONENT LIBRARY LOCATION
 /// https://web.dev/design-system/component/figure
 figure {
-  border: 1px solid var(--color-stroke);
-  padding: 1em;
-  border-radius: $global-radius-base;
+  display: block;
+  width: 100%;
```

Comment:
The img and caption align in "pure" figure is still not great:
<img width="1099" alt="Screenshot 2022-01-18 at 13 35 04" src="https://user-images.githubusercontent.com/1914261/149938900-7391c49b-adc0-4453-b44e-e8ced40123ed.png">

How about you add "text-align: center;" to the non-sized and non-floated figurecaption? The outcome should be:

<img width="675" alt="Screenshot 2022-01-18 at 13 40 38" src="https://user-images.githubusercontent.com/1914261/149939182-1befd1f8-d4e9-44a3-8952-30efc00c6618.png">